src/main/java/com/kramekk1/medicalClinic/PatientController.java

```diff
@@ -0,0 +1,41 @@
+package com.kramekk1.medicalClinic;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/patients")
+public class PatientController {
+
+    private final PatientRepository patientRepository;
+    private final PatientService patientService;
+
+    @GetMapping("/all")
+    public List<Patient> getPatients() {
+        return patientRepository.getPatients();
+    }
+    @GetMapping("/{email}")
+    public Optional<Patient> getPatientByEmail(@PathVariable String email) {
+        return patientRepository.findPatientByEmail(email);
```

Comment:
Do rozwiniecia warstwa serwisów -> to ona powinna przetwarzać zapyatnia a kontroller je odbierać